src/constants/fieldTypesNew.js

```diff
@@ -9,6 +9,7 @@ const fieldTypes = {
   latlong: 'latlong', // [float, float] defines [lat, lng]
   date: 'date',
   daterange: 'daterange', // [date, date]
+  specifiedTime: 'specifiedTime', // { time: string, timePrecision: string }
```

Comment:
This file name tho.

Reply:
yeahh i still need to delete the old one!!!

Reply:
it's on my todo list...